### PR TITLE
analytics

### PR DESCRIPTION
**Pull request by mid998**

### CODE REVIEW

File: antora-playbook.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ site:
   url: https://red-hat-storage.github.io/ocs-sop
   start_page: sop::index.adoc
   robots: allow
+  keys:
+    google_analytics: 'G-LGCEEZGN54'
```
Comment (mulbc) [inline antora-playbook.yml:7]: You copied the ID from ocs-training... please use your own ID `G-PDMKGVZNGE`

**Review by mulbc — CHANGES_REQUESTED** [review]
Please use the ocs-sop GA ID

File: antora-playbook.yml (hunk 1 above)
Comment (mid998) [reply antora-playbook.yml]: Just waking up... fixed.

**Review by mulbc — APPROVED** [review]
lgtm

**Pull request merged**